src/fake-db.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "title": "Snow Crash",
```

Comment:
Some stuff I copied off of wikipedia. You'll come across the .json format approximately one million times a day doing web development.

It's a convenient way of storing data as text. Probably all major programming languages have tools to parse it and use it.

Reply:
Looks good. I'm familiar with json format for JS arrays, though I haven't reached that point in the Odin material.